nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -292,28 +301,14 @@ def _get_bias_shift(self,
         raw_output = engine.infer(input_blob)
         q_outputs = self._backend_entity.process_model_output(raw_output, output_name)
```

Comment:
Should we split this function into two? The first is common with obtaining `raw_output` and the second one is backend specific with returning `bias_shift`?

Reply:
Not sure